src/Infrastructure/IO/tests/makefile

```diff
@@ -81,3 +84,13 @@ RUN_ESMF_IO_YAMLUTestUNI:
 RUN_ESMF_IO_MultitileUTest:
 	rm -f $(ESMF_TESTDIR)/ESMF_IO_MultitileUTest*.nc
 	$(MAKE) TNAME=IO_Multitile NP=8 ftest
+
+# Note that we run this with NP=1 because there is little value in running these tests
+# with multiple processors
+RUN_ESMF_IO_FileTypeCheckUTest:
+	cp -f filetypecheck_data/*.nc $(ESMF_TESTDIR)
+	$(MAKE) TNAME=IO_FileTypeCheck NP=1 ftest
```

Comment:
There didn't seem to be value in running this unit test with multiple processors, so I set it up to always run with a single processor. Is this a reasonable thing to do?